exchange/response_test.go

```diff
@@ -29,10 +29,13 @@ import (
 	"github.com/google/webpackager/resource/preload"
 )
 
-func makeHTTPResponse(payload []byte) *http.Response {
+func makeHTTPResponse(t *testing.T, payload []byte) *http.Response {
 	rw := httptest.NewRecorder()
 	rw.WriteHeader(200)
-	rw.Write(payload)
+	_, err := rw.Write(payload)
```

Comment:
I'm not sure if we should inspect the error from Write. io.Writer.Write returns (int, error) since it's presumably designed with file and network I/O in mind thus follows write(2) as in POSIX API, but many Write methods are declared as such only to satisfy the contract and never return a non-nil error. In fact this code looks weird: we don't check the error for rw.WriteHeader() (in fact we can't) but do for rw.Write().

Reply:
Done 3x.